src/action.js

```diff
@@ -33,8 +33,60 @@ function log(message) {
   console.log(`[${new Date().toUTCString()}] ${message}`);
 }
 
+function pad(n) {
+    var s = '000' + n;
+    return s.substring(s.length-4);
```

Comment:
nit: Although unlikely to hit more than 4 digits anytime soon, maybe we can still prevent this from being an error if that ever did happen? e.g. one option would be to just return `n` as is if it's greater than 4 already